api/index.js

```diff
@@ -20,6 +20,7 @@ export default async (req, res) => {
     hide_rank,
     show_icons,
     include_all_commits,
+    year,
```

Comment:
Current query option name sounds like it should affect on the whole stats instead of commits only. What do you guys think about renaming it into `commits_year`? Or may be anyone have better suggestions?

Reply:
Agreed. Will change to `commits_year`.